scripts/generate_descriptions.py

```diff
@@ -0,0 +1,116 @@
+"""Generate descriptions.json for the dashboard."""
+
+import pathlib  # noqa: I001
```

Comment:
Why is it complaining about unsorted imports and why can't we just sort it?

Reply:
fixed